viewer/views/layout.jade

```diff
@@ -159,6 +159,13 @@ html
         - if (molochClusters)
           each key in Object.keys(molochClusters)
             command.sendSessionButton(cluster=key,label="Send Session to #{molochClusters[key].name}")
+        - if (user.fragments && user.fragments.query)
+          - var keys = Object.keys(user.fragments.query).sort()
+          - if (titleLink == "sessionsLink" && keys.length > 0)
+            span 
+              b Reports
+            each key in keys
+              command#queryReport(label="#{user.fragments.query[key].name}", data-filename="#{user.fragments.query[key].identifier}")
```

Comment:
when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
command(class="queryReport", label="#{user.fragments.query[key].name}", data-filename="#{user.fragments.query[key].identifier}")